README.md

```diff
@@ -1,6 +1,12 @@
 # Darkdetect
 
-This package allows to detect if the user is using Dark Mode ([macOS 10.14+](https://support.apple.com/en-us/HT208976) or [Windows 10 1607+](https://blogs.windows.com/windowsexperience/2016/08/08/windows-10-tip-personalize-your-pc-by-enabling-the-dark-theme/)). The main application of this package is to detect the Dark mode from your GUI Python application (Tkinter/wx/pyqt/qt for python (pyside)/...) and apply the needed adjustments to your interface. Darkdetect is particularly useful if your GUI library **does not** provide a public API for this detection (I am looking at you, Qt). In addition, this package does not depend on other modules or packages that are not already included in standard Python distributions.
+This package allows to detect if the user is using Dark Mode on:
+
+- ([macOS 10.14+](https://support.apple.com/en-us/HT208976)
+- [Windows 10 1607+](https://blogs.windows.com/windowsexperience/2016/08/08/windows-10-tip-personalize-your-pc-by-enabling-the-dark-theme/))
+- Linux with [a dark GTK theme](https://www.gnome-look.org/browse/cat/135/ord/rating/?tag=dark) on an **experimental** basis (see notes below)
```

Comment:
IMHO, no need to put experimental in bold... After all, this package is still on version 0.x, it's all "experimental".